kratos/utilities/time_discretization.h

```diff
@@ -0,0 +1,113 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+#if !defined(KRATOS_TIME_DISCRETIZATION_H_INCLUDED )
+#define  KRATOS_TIME_DISCRETIZATION_H_INCLUDED
+
+// System includes
+#include <vector>
+
+// External includes
+
+// Project includes
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+class BDF1
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class BDF2
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class BDF3
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class BDF4
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class BDF5
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class BDF6
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class Newmark
+{
+public:
+    double GetBeta() const { return 0.25; }
+    double GetGamma() const { return 0.5; }
+};
+
+class Bossak
+{
+public:
+    Bossak(const double AlphaM)
+        : mAlphaM(AlphaM) {}
+    double GetBeta() const { return 0.5 + mAlphaM; }
+    double GetGamma() const { return 0.25 * (1+mAlphaM) * (1+mAlphaM); }
```

Comment:
Probably we should also have GetAlphaM()

Reply:
done